theme/account/resources/public/layout.css

```diff
@@ -1,271 +1,267 @@
+  --pf-global--FontSize--sm: 0.875rem;
+  --pf-global--FontSize--xs: 0.75rem;
+  --pf-global--FontWeight--light: 300;
+  --pf-global--FontWeight--normal: 400;
+  --pf-global--FontWeight--semi-bold: 500;
+  --pf-global--FontWeight--bold: 600;
+  --pf-global--FontWeight--redhatfont--bold: 700;
+  --pf-global--LineHeight--sm: 1.3;
+  --pf-global--LineHeight--md: 1.5;
+  --pf-global--ListStyle: disc outside;
+  --pf-global--Transition: all 250ms cubic-bezier(0.42, 0, 0.58, 1);
+  --pf-global--TimingFunction: cubic-bezier(0.645, 0.045, 0.355, 1);
+  --pf-global--TransitionDuration: 250ms;
+  --pf-global--arrow--width: 0.9375rem;
+  --pf-global--arrow--width-lg: 1.5625rem;
+  --pf-global--target-size--MinWidth: 44px;
+  --pf-global--target-size--MinHeight: 44px;
 }
 
-
 .brand {
-    height: 35px;
+  height: 35px;
 }
 
 .delete-button {
-    width: 150px;
-    height: 50px;
+  width: 150px;
+  height: 50px;
 }
 
 @media (max-width: 320px) {
-    .delete-button {
-        width: 120px;
-        height: 50px;
-    }
-
+  .delete-button {
+    width: 120px;
+    height: 50px;
+  }
 }
 
 @font-face {
-    font-family: "Montserrat";
-    font-style: normal;
-    font-weight: 400;
-    src:
-            local("Montserrat Regular"),
-            local("Montserrat-Regular"),
-            url("../fonts/Montserrat-Regular.woff2") format("woff2"),
-            url("../fonts/Montserrat-Regular.woff") format("woff");
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 400;
+  src: local("Montserrat Regular"), local("Montserrat-Regular"),
+    url("../fonts/Montserrat-Regular.woff2") format("woff2"),
+    url("../fonts/Montserrat-Regular.woff") format("woff");
 }
 
 @font-face {
-    font-family: "Montserrat";
-    font-style: normal;
-    font-weight: 700;
-    src:
-            local("Montserrat Bold"),
-            local("Montserrat-Bold"),
-            url("../fonts/Montserrat-Bold.woff2") format("woff2"),
-            url("../fonts/Montserrat-Bold.woff") format("woff");
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 700;
+  src: local("Montserrat Bold"), local("Montserrat-Bold"),
+    url("../fonts/Montserrat-Bold.woff2") format("woff2"),
+    url("../fonts/Montserrat-Bold.woff") format("woff");
 }
 
 @font-face {
-    font-family: "Montserrat";
-    font-style: normal;
-    font-weight: 900;
-    src:
-            local("Montserrat Black"),
-            local("Montserrat-Black"),
-            url("../fonts/Montserrat-Black.woff2") format("woff2"),
-            url("../fonts/Montserrat-Black.woff") format("woff");
-}
-
-#basic-authentication-categ-title {
-    display: none;
-}
-
-#basic-authentication-categ-title + .pf-c-data-list{
-    display: none;
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 900;
+  src: local("Montserrat Black"), local("Montserrat-Black"),
+    url("../fonts/Montserrat-Black.woff2") format("woff2"),
+    url("../fonts/Montserrat-Black.woff") format("woff");
```

Comment:
Is the change in indentation of the  whole block intentional?
Shouldn't that be two commits, one for the indentation and one for removing the "display:  none"?

Reply:
No, it was not intentional. Thanks for the feedback.